clients/vscode-hlasmplugin/README.md

```diff
@@ -323,6 +318,25 @@ You can also chain the preprocessors in the following way:
 }
 ```
 
+## Download Dependencies
+
+You can use HLASM Language Support to download dependencies specified in mainframe data sets in `proc_grps.json` to your workspace. To connect to the mainframe, we recommend setting up a `zowe zosmf` or `zowe zftp` profile to store your credentials and connection information.
+
+1. Press **F1** to open the Command Pallet.
+2. Run the commend **HLASM: Download dependencies**.
+3. Do one of the following:
+   - Specify the name of a `zowe zosmf` or `zowe zftp` profile that contains your mainframe credentials and server information.
```

Comment:
The profile name needs to be prefixed with `@` that's how the command distinguishes profile name from host name...